src/sentry/api/endpoints/organization_sdk_updates.py

```diff
@@ -70,7 +70,7 @@ def serialize(data, projects):
 
 
 @region_silo_endpoint
-class OrganizationSdkUpdatesEndpoint(OrganizationEventsEndpointBase):
+class OrganizationSdkUpdatesEndpoint(OrganizationEndpoint):
```

Comment:
Now that `handle_query_errors` has been decoupled from the `OrganizationEventsEndpointBase`, we can extend the `OrganizationEndpoint` instead